src/DotNetProjectFile.Analyzers/tools/uninstall.ps1

```diff
@@ -0,0 +1,10 @@
+param($installPath, $toolsPath, $package, $project)
+
+$analyzersPath = Split-Path -Path $toolsPath -Parent
+$analyzersPath = Join-Path $toolsPath "analyzers"
+$analyzerFilePath = Join-Path $analyzersPath "DotNetProjectFile.Analyzers.dll"
+try {
```

Comment:
Why the try catch if you don't do anything with the catch?

Reply:
This is what Microsoft suggests. So I just followed that:

https://learn.microsoft.com/en-us/nuget/guides/analyzers-conventions#install-and-uninstall-scripts